Userland/Libraries/LibHTTP/Job.cpp

```diff
@@ -60,14 +57,7 @@ static Optional<ByteBuffer> handle_content_encoding(ByteBuffer const& buf, Depre
             // "Note: Some non-conformant implementations send the "deflate"
             //        compressed data without the zlib wrapper."
             dbgln_if(JOB_DEBUG, "Job::handle_content_encoding: Zlib::decompress_all() failed. Trying DeflateDecompressor::decompress_all()");
-            auto uncompressed_or_error = Compress::DeflateDecompressor::decompress_all(buf);
-
-            if (uncompressed_or_error.is_error()) {
-                dbgln("Job::handle_content_encoding: DeflateDecompressor::decompress_all() failed: {}", uncompressed_or_error.error());
-                return {};
-            }
-
-            uncompressed = uncompressed_or_error.release_value();
+            return TRY(Compress::DeflateDecompressor::decompress_all(buf));
```

Comment:
This should probably be:
```c++
uncompressed = TRY(Compress::DeflateDecompressor::decompress_all(buf));
```

Otherwise we skip the debug prints below.

Reply:
wow, nice catch !